### PR TITLE
* Added java 17 to available JDKs

### PR DESCRIPTION
**Pull request by philiplourandos**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,6 +5,8 @@ RUN apk --no-cache add openjdk11 --repository=http://dl-cdn.alpinelinux.org/alpi
 RUN apk --update add openjdk14-jre --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
 RUN apk --update add openjdk15-jre --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
 RUN apk --update add openjdk16-jre --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
+RUN apk --update add openjdk17 --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```
Comment (qcastel) [inline Dockerfile:8]:
```suggestion
RUN apk --update add openjdk17-jre --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

what do you think? I already did jre but perhaps I was wrong from the start

Comment (philiplourandos) [reply Dockerfile:8]: My understanding is the JRE does not have compiler tools etc. For running a java app the JRE is preferred, but if we are building a repo I'd think the JDK is preferable...?

Comment (qcastel) [reply Dockerfile:8]: Yes, I agree too. Not sure why thought it worked with jre then.

Well, for alignement purposes, I will switch all of them to JDK

**Comment by qcastel** [comment]
I actually push the change and will propagate it to the github action

**Pull request closed without merging**